src/image/bigcommerce.ts

```diff
@@ -1,5 +1,15 @@
-import type { BaseTransformerProps } from "./types"
-import { BigCommerceUrlGroups } from "./types"
+import type { Operations } from "unpic/types"
+
+type BigCommerceUrlGroups = {
```

Comment:
moved bigcommerce regex type defintion here, since it is not used outside of this module